### ISSUE TITLE
Specify a timeout to the requests.get call

### ISSUE THREAD

**Issue by Kraymer** [dgilland/omdb.py#9]
I noticed that call to `omdb.get()` sometimes hung forever.
Why not specify a `timeout=` parameter to the underlying `requests.get()` call ?

**Comment by dgilland** [comment]
Good idea!

I've implemented this feature in `v0.6.0`: https://pypi.python.org/pypi/omdb/0.6.0

```python
import omdb

# Set timeout per function call
omdb.get(title='some movie', timeout=5)

# Set global default
omdb.set_default('timeout', 5)

# Default of 5 seconds will be used.
omdb.get(title='some movie')

# Disable or override the timeout per function call
omdb.get(title='some movie', timeout=None)
omdb.get(title='some movie', timeout=10)
```

**Comment by Kraymer** [comment]
Perfect!
